Added README (#5)
bow-swift/nef-plugin: README.md

<p align="center">
    <img src="https://github.com/bow-swift/bow-art/blob/master/assets/nef-brand-xcode-shadow.png?raw=true" alt="nef: an Xcode 11 plugin" width="30%"/>
</p>

<p align="center">
    <a href="https://github.com/bow-swift/nef">
    <img src="https://img.shields.io/badge/nef-%E2%9C%93-blueviolet" alt="nef">
    </a>
    <a href="https://swift.org/download">
    <img src="https://img.shields.io/badge/swift-5-ED523F.svg?style=flat" alt="Swift 5">
    </a>
    <a href="https://developer.apple.com/xcode/swiftui">
    <img src="https://img.shields.io/badge/Swift%20UI-%E2%9C%93-orange" alt="Swift UI">
    </a>
    <a href="https://developer.apple.com/xcode">
    <img src="https://img.shields.io/badge/IDE-Xcode%2011-blue" alt="Xcode 11">
    </a>
</p>

&nbsp;

> It is a plugin for Xcode to integrate some of the nef's features directly in the IDE. Using the core of `nef`, you can [export snippets](https://github.com/bow-swift/nef#-exporting-carbon-code-snippets) from your code selection directly in Xcode.

Reviewer: ```suggestion
This project provides a plugin for Xcode to integrate some nef features directly in the IDE. Using the core of `nef`, you can [export snippets](https://github.com/bow-swift/nef#-exporting-carbon-code-snippets) from your code selection directly in Xcode.
```
